examples/build.gradle

```diff
@@ -0,0 +1,30 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.3'
+    }
+}
+
+ext {
```

Comment:
`default` doesn't feel like the right term. Do we ever _not_ use the default? Maybe just remove it entirely